cluser_validation_checklist.md

```diff
@@ -0,0 +1,87 @@
+# Cluster Validation Checklist
+## Check general cluster status
```

Comment:
What file in this repo stores the results of the checklist validations?

Reply:
Do we want something automated here? If not this checklist can be copied and filled out.